<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardWindow</class>
 <widget class="QWidget" name="CardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>311</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>429</width>
    <height>311</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>429</width>
    <height>311</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SGuard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/Resources/Icons/SGuard.png</normaloff>:/icons/Resources/Icons/SGuard.png</iconset>
  </property>
  <widget class="QLineEdit" name="CardName">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>50</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="URL">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>100</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Login">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>150</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>200</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Card Info</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>30</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>80</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>130</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>180</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TrashButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/Icons/trash.png</normaloff>:/icons/Resources/Icons/trash.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="EyeButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/Icons/eye.png</normaloff>:/icons/Resources/Icons/eye.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CircleButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/Icons/circle.png</normaloff>:/icons/Resources/Icons/circle.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/Icons/cancel.png</normaloff>:/icons/Resources/Icons/cancel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Generator Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Copy">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/Icons/154525.png</normaloff>:/icons/Resources/Icons/154525.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>13</width>
     <height>13</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Generator">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPass">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Copy_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/Icons/154525.png</normaloff>:/icons/Resources/Icons/154525.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>13</width>
     <height>13</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Copy_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/Icons/154525.png</normaloff>:/icons/Resources/Icons/154525.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>13</width>
     <height>13</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Copy_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/Icons/154525.png</normaloff>:/icons/Resources/Icons/154525.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>13</width>
     <height>13</height>
    </size>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>272</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/Icons/previous.png</normaloff>:/icons/Resources/Icons/previous.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblLogin">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblURL">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Password</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CardWindow</receiver>
   <slot>Check_Password(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Login</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CardWindow</receiver>
   <slot>Check_Login(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>URL</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CardWindow</receiver>
   <slot>Check_URL(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CardName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CardWindow</receiver>
   <slot>Check_Name(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Check_Password(QString)</slot>
  <slot>Check_Login(QString)</slot>
  <slot>Check_URL(QString)</slot>
  <slot>Check_Name(QString)</slot>
 </slots>
</ui>
